x-pack/legacy/plugins/infra/public/containers/logs/log_highlights/redux_bridges.tsx

```diff
@@ -8,23 +8,11 @@ import React, { useEffect, useContext } from 'react';
 
 import { TimeKey } from '../../../../common/time';
 import { withLogFilter } from '../with_log_filter';
-import { withStreamItems } from '../with_stream_items';
 import { withLogPosition } from '../with_log_position';
 import { LogHighlightsState } from './log_highlights';
 
 // Bridges Redux container state with Hooks state. Once state is moved fully from
 // Redux to Hooks this can be removed.
```

Comment:
I guess this can go away now 🎉

Reply:
Not quite yet, after #50397 and #50398 these can all go away